### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by haliphax** [haliphax/stream-avatars#6]
Documentation is needed both for developing new avatars within the system and getting the chat bot set up with OAuth bearer tokens, etc.

**Comment by Sky84** [comment]
Yup ! I personnally need this ! I've set up the bot and everything but I dont know where I need to go, to see the avatars

**Comment by haliphax** [comment]
Should be https://localhost:5000/avatars/static/index.html, though the avatars system is being completely rewritten from scratch right now to use PhaserJS and will soon be a bit different.

**Comment by haliphax** [comment]
The avatars system is going to be developed further under a standalone project. It's mostly ready, just needs more sprites added. It's already got feature parity with the current system, but needs no back-end code and runs entirely from static files. It still needs to be served over HTTP/S for `fetch` and dynamic `import` to work, but there's no code running on the server.

I have plans for splitting out the chat overlay in a similar fashion. By that point, there won't be much left of the bot at all. I will likely retire this project and just build front-end-only dynamic overlays that can be integrated with something like [TAU](https://github.com/Team-TAU/tau).

Here's the new avatar system's home: https://github.com/haliphax/hxavatars
